pods/covid-spain-chart/espana-covid-marzo.container.tsx

```diff
@@ -0,0 +1,12 @@
+import { EspanaCovidMarzoComponent } from "./espana-covid-marzo.component";
+import React from "react";
+
+interface Props {
+}
+
+export const EspanaCovidMarzoContainer: React.FC<Props> = (props: Props) => {
+  // This container will show the generated map
+  return (
```

Comment:
Mmm.. the purpose of the container is to handle all the data loading and business (in some scenarios this can become an antipattern).

IMHO we should add here the method to load the data, then pass it down as a prop to the component, and the createChart method in the d3 should receive this as a param.

In case we had an scenario with a lot of subcomponents we could make use of React.context.